themes/solr/templates/operator/features.html

```diff
@@ -0,0 +1,201 @@
+          <div class="content">
+            <h3>Customize the Kubernetes Components</h3>
+            <p>The Solr Operator gives many options for customizing the Kubernetes components used to deploy the Solr Cloud.</p>
+          </div>
+        </div>
+      </li>
+    </ul>
+  </div>
+</section>
+
+<section class="hero alternate" id="solr-metrics">
+  <div class="row">
+    <div class="large-12 columns">
+      <div class="annotation">
+        Solr Prometheus Exporter
+      </div>
+      <h1>
+        Managing Solr Metrics in Kubernetes.
+      </h1>
+      <div class="down-arrow"><a data-scroll href="#solr-metrics-features"><i class="fa fa-angle-down fa-2x red"></i></a></div>
+    </div>
+  </div>
+</section>
+
+<section class="gray col-4 offset-small" id="solr-metrics-features">
+  <div class="row">
+    <ul class="small-block-grid-1 medium-block-grid-3">
+      <li>
+        <div class="box">
+          <div class="img"><img src="{{ SITEURL }}/theme/images/Solr_Icons_standards_based_open_interfaces.svg"/></div>
+          <div class="content">
+            <h3>Automatically Update Information</h3>
+            <p>Automatically restart when updates to TLS secrets, Auth or ZooKeeper Connection Strings are made.</p>
+          </div>
+        </div>
+      </li>
+      <li>
+        <div class="box">
+          <div class="img"><img src="{{ SITEURL }}/theme/images/Solr_Icons_extensible_plugin.svg"/></div>
+          <div class="content">
+            <h3>Integrate with Prometheus Stack</h3>
+            <p>Integration with the Prometheus Operator in Kubernetes is easy and well documented.</p>
+          </div>
+        </div>
+      </li>
+      <li>
+        <div class="box">
+          <div class="img"><img src="{{ SITEURL }}/theme/images/Solr_Icons_flexible_and_adaptable.svg"/></div>
+          <div class="content">
+            <h3>Customize the Kubernetes Components</h3>
+            <p>The Solr Operator gives many options for customizing the Kubernetes components used to deploy the Solr Prometheus Exporter.</p>
+          </div>
+        </div>
+      </li>
+    </ul>
+  </div>
+</section>
+
+<section class="hero alternate" id="solr-backup">
+  <div class="row">
+    <div class="large-12 columns">
+      <div class="annotation">
+        Solr Backups
+      </div>
+      <h1>
+        Backup your SolrClouds in Kubernetes.
+      </h1>
+      <div class="down-arrow"><a data-scroll href="#solr-backup-features"><i class="fa fa-angle-down fa-2x red"></i></a></div>
+    </div>
+  </div>
+</section>
+
+<section class="gray col-4 offset-small" id="solr-backup-features">
+  <div class="row">
+    <ul class="small-block-grid-1 medium-block-grid-3">
+      <li>
+        <div class="box">
+          <div class="img"><img src="{{ SITEURL }}/theme/images/Solr_Icons_an_ajax_based.svg"/></div>
+          <div class="content">
+            <h3>Multiple Storage Options</h3>
```

Comment:
There is a mouse-over effect for each of the feature blocks. But no link.

Could it be an idea to link to the part of the github Wiki documentation for each of the features?

Reply:
Yes, I think that would be a great idea.